composable/assetlist.json

```diff
@@ -229,6 +229,359 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/tinkernet/images/tnkr.svg"
       }
+    },
+    {
+      "description": "The native token of Ethereum, bridged via IBC.",
+      "denom_units": [
+        {
+          "denom": "wei",
```

Comment:
for all ibc/ assets, the denom for the exp: 0 unit must be the ibc denom, with anything else as a n alias.
e.g., 
```
{
  "denom": "ibc/F9D075D4079FC56A9C49B601E54A45292C319D8B0E8CC0F8439041130AA7166C",
  "exponent": 0,
  "aliases": [
    "wei"
  ]
},
```